samples/bootloader/src/main.c

```diff
@@ -46,14 +46,15 @@ extern u32_t _vector_table_pointer;
 
 static void boot_from(const struct fw_info *fw_info)
 {
-	u32_t *vector_table = (u32_t *)fw_info->firmware_address;
+	u32_t *vector_table = (u32_t *)fw_info->image_address;
 
 	printk("Attempting to boot from address 0x%x.\n\r",
-		fw_info->firmware_address);
+		fw_info->image_address);
 
-	if (!bl_validate_firmware_local(fw_info->firmware_address,
+	if (!bl_validate_firmware_local(fw_info->image_address,
```

Comment:
bl_validate_firmware_local is not documented (but not part of this PR)

I would suggest indicating that fw_info_invalidate is required to be run when this is false (as this function is also used by prevalidate so FW can't be invalidated directly)

Reply:
Added dox for `bl_validate_firmware_local()`

Reply:
Looks good